DOCSP-29676 v5.4 update (#675)
mongodb/docs-node: source/whats-new.txt

* :ref:`Version 3.7 <version-3.7>`
* :ref:`Version 3.6 <version-3.6>`

.. _version-5.4:

What's New in 5.4
-----------------

New features of the 5.4 {+driver-short+} release include:

.. important:: Deprecation Notice

>   - The ``collStats`` helper is deprecated. Use the :manual:`$collStats

Reviewer: S: Looking at the [JIRA ticket](https://jira.mongodb.org/browse/NODE-4970), `collStats` is a helper command. Might be useful to specify here.

```suggestion
  - The ``collStats`` helper command is deprecated. Use the :manual:`$collStats
```

    </reference/operator/aggregation/collStats>` aggregation operator instead.

>   - Methods of ``db.command()`` that were incorrectly included in the
>     typescript interface have been deprecated.

Reviewer: I: I think these were options, not methods.

```suggestion
  - The TypeScript interface passed to the ``db.command()`` method incorrectly
  includes certain options. These options have been deprecated.
```

Author: Oops, good catch. Changed

> - The ``ChangeStream.tryNext`` method now uses the ``TChange`` schema

Reviewer: ```suggestion
- The ``ChangeStream.tryNext()`` method now uses the ``TChange`` schema
```

> - The ``ChangeStream.tryNext`` method now uses the ``TChange`` schema
>   generic instead of the untyped ``Document`` interface.

Reviewer: S: Something like:

```suggestion
- The ``ChangeStream.tryNext`` method now uses the schema-specific
``TChange`` generic type instead of the ``Document`` interface.
```
